extmod/modframebuf.c

```diff
@@ -536,6 +536,10 @@ static mp_obj_t framebuf_ellipse(size_t n_args, const mp_obj_t *args_in) {
     } else {
         mask |= ELLIPSE_MASK_ALL;
     }
+    if (args[2] == 0 && args[3] == 0) {
+        setpixel_checked(self, args[0], args[1], args[4], 1);
```

Comment:
I think the last argument should be `mask`, in case the user specifies no fill and no quadrants (in which case nothing should be drawn).

Reply:
Thanks, I've fixed this - it should also draw nothing if the user specifies fill and no quadrants, I think, so the last argument should actually be `mask  & ELLIPSE_MASK_ALL` since the 5th bit is for fill/no-fill.

Also I've added a test for this case.